app/jobs/dashboard/applicant_email_job.rb

```diff
@@ -2,9 +2,13 @@ module Dashboard
   class ApplicantEmailJob < ActiveJob::Base
     include SuspendableJob
 
+    APPLY_EMAIL_REGEX = /#{Rails.configuration.x.email_domain.suffix}/.freeze
+
     def perform(application)
       return if job_suspended?
 
+      return if application.applicant.email.match?(APPLY_EMAIL_REGEX) && Rails.env.test?
```

Comment:
Won't this change mean that it will still do it on Prod? 
How about 
```ruby
%w[production test].include?(Rails.env)
```

Reply:
I added .test? just for UAT testing purposes, i've removed that change now.